portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.servlets;
+
+public class Comment {
+  long id;
+  String text;
+  String timestamp;
```

Comment:
I suggest to add the visibility, e.g. private final long id; The same for the other fields

Reply:
Fixed!